parsing-scripts/parse_build_history.py

```diff
@@ -0,0 +1,124 @@
+import os
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "MacPorts.settings")
+
+import django
+django.setup()
+
+from ports.models import BuildHistory
+import urllib.request
+import ssl
+import json
+import datetime
+
+
+builders = [
+    '10.14_x86_64',
+    '10.13_x86_64',
+    '10.12_x86_64',
```

Comment:
If these values are hardcoded, they should probably be hardcoded in some separate settings file, to avoid potentially repeating yourself in every script where you might need this list.

Reply:
Yes, I will be creating a table of builders, which can be updated as newer versions of macOS get released.